docs/release-notes/13-2-0/README.md

```diff
@@ -3,22 +3,87 @@ title: OpenProject 13.2.0
+
+See our user guide to learn more about this Enterprise add-on and how to [use the SharePoint integration](../../user-guide/file-management/one-drive-integration/).
+
+## Show file links of files that are not available to the user in the cloud storage
+
+In 13.2 we improved the functionality of file storages. Even if a file has been deleted on a file storage, it will still be displayed under the Files tab in OpenProject work packages, allowing users to better keep track of project files. 
+
+![Deleted file storage file in OpenProject](deleted-file.png)
+
+This new feature is available for [both file storage integrations](../../user-guide/file-management). 
+
+## Filter for roles, groups, and shares in the project members list
+With OpenProject 13.2, users can filter the list of project member based on user roles, groups, and shared work packages. This enables project members to promptly identify others with specific roles or individuals outside the project team who have  gained access through the newly introduced Sharing feature. 
+
+![Project members filter in OpenProject](project-members.png)
+
+Read more about filtering the project members list in our [user guide](../../user-guide/members/#project-members-overview). 
+
+## Allow assignee and accountable for shared work packages (Enterprise add-on)
+
+In OpenProject 13.1, we introduced the work package sharing feature with external users. In the 13.2 release, these shared users can now be designated as assignees and accountable for the work packages that have been shared with them. This is useful for teams collaborating with external partners who cannot access sensitive project data but still require access to specific tasks within the project.
+
+Learn more about our [Sharing work packages (Enterprise add-on)](../../user-guide/work-packages/share-work-packages/).
+
+## Status change without rights to edit a work package
+
+In OpenProject 13.2, the ability to modify the status of a work package is now separated from the broader "Edit work package" permission. This separation means that a user can be granted the right to change the status without having the permission to edit the entire work package.
+
+Read more about [roles and permissions for users of OpenProject](../../system-admin-guide/users-permissions/roles-permissions/).
+
+## Quick context menu in Gantt view: Show relations
+
+Quick context menu in the Gantt view now includes "Show relations" option. It displays all existing relations for the selected work package. 
+
+Please note that these option is only available if you have selected a Gantt view (i.e. not a table or cards view). 
+
+![Quick context menu in Gantt charts in OpenProject](gantt-relations.png)
+
+Read more about [Gantt charts in OpenProject](../../docs/user-guide/gantt-chart/)
+
+## New field names and calculation of work and remaining work
+Several field names were changed in OpenProject 13.2.
+
+- **Estimated time** is now called Work
```

Comment:
I would emphasize the new terms, not the old ones. So:

- Estimated time is now called **Work**
- Remaining hours is now called **Remaining work**
- Progress (%) is now called **% Complete**